scripts/functions/requirements/ubuntu

```diff
@@ -99,23 +99,32 @@ requirements_debian_define()
 
     (rbx*head|rubinius*head)
       requirements_check_fallback git git-core
-      requirements_ubuntu_libs_default clang-3.5 llvm-3.5 llvm-3.5-dev
+      requirements_ubuntu_libs_default clang-3.5 llvm-3.5 llvm-3.5-dev libssl-dev
       rvm_configure_flags+=( --cc=clang --cxx=clang++ )
       ;;
 
     (rbx*|rubinius*)
-      requirements_ubuntu_libs_default clang-3.5 llvm-3.5 llvm-3.5-dev libedit-dev
+      requirements_ubuntu_libs_default clang-3.5 llvm-3.5 llvm-3.5-dev libedit-dev libssl-dev
       rvm_configure_flags+=( --cc=clang --cxx=clang++ )
       ;;
 
+    (ruby-2.3*|ruby-2.2*|ruby-2.1*|ruby-2.0*|ruby-1.9*)
+      if [[ "${_system_name}" == "Debian" && ("${_system_version}" = "9" || "${_system_version}" > "9") ]]
```

Comment:
bash does not recognize strings or numbers, if the code name is `9` it should work just fine :) ... unless it will get a code name later on?